src/graphql/data/TopTokens.ts

```diff
@@ -135,16 +130,31 @@ interface UseTopTokensReturnValue {
   sparklines: SparklineMap
 }
 
+function replaceWrappedWithNative<T extends { address: string | null } | null>(chainId: number, token: T): T {
```

Comment:
I feel like this util should be somewhere else. I get the sense from the (admittedly sparse) Relay docs that this file is intended to be scoped specifically to queries and maaaybe their primary access hooks. Utils related to the data from those queries maybe should go in `src/graphql/utils/<Query>`?